thorlcr/activities/msort/thsortu.cpp

```diff
@@ -294,44 +292,44 @@ class CJoinHelper : implements IJoinHelper, public CSimpleInterface
     OwnedConstThorRow denormLhs;
     RtlDynamicRowBuilder denormTmp;
     CThorExpandingRowArray denormRows;
-    unsigned denormCount;
-    unsigned joinCounter;
-    size32_t outSz;
-    unsigned rightidx;
-    enum { JScompare, JSmatch, JSrightgrouponly, JSonfail } state;
-    bool eofL;
-    bool eofR;
+    unsigned denormCount = 0;
+    unsigned joinCounter = 0;
+    size32_t outSz = 0;
+    unsigned rightidx = 0;
+    enum { JScompare, JSmatch, JSrightgrouponly, JSonfail } state = JScompare;
+    bool eofL = false;
+    bool eofR = false;
     IHThorJoinArg *helper;
```

Comment:
ok, why not ?

Reply:
it's always initialized by the ctor, but still I prefer all member declarations to be initialized.
I'll fix.

Reply:
As is kind, which is only reasone why I asked - it seemed to be the one left out.